### ISSUE TITLE
Tyre texture going backwards & mat is reflective

### ISSUE THREAD

**Issue by LocoCZ** [SmartlyDressedGames/Unturned-3.x-Community#4555]
<details><summary>Texture</summary>
It is not quite visible, but I do have an eye for motorsport related things. The tire detailing seems to point backwards rather than to the front.

![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/81763752/9a65bf69-9866-47ca-a37a-d772bfbd4aca)
If I provide some examples of some tyres, you can see their detailing goes to the front:
<details><summary>Tyre examples</summary>

![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/81763752/68714115-c53e-41f3-8c3d-e537ab0bc9ea)

![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/81763752/fb836061-230f-45d1-bd20-3a966336f37d)
</details>
</details>

<details><summary>Reflections</summary>
Another thing I would like to mention, tyres are not really reflective and I would tick off reflections for the tyre material in Unity. 

![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/81763752/a6f25131-479a-426e-a357-95ce283f1087)
The white spots are clouds. They look immediately better without reflections:

![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/81763752/bbd74d35-bb24-4f82-912b-1e112519066a)
</details>

**Comment by SDGNelson** [comment]
Thanks for the reports.

Perhaps racecars are different - I didn't make a tire model specifically for them I'm afraid - but when I was looking at a variety of tires to reference for the new models it seemed like it varied per design. i.e., as far as I could tell it wasn't like *all* tires have grooves pointing a certain direction. Do you know of an informational page that describes otherwise?

As for reflections, the tire material has Metallic 0 and Smoothness 0, but all materials have some amount of specular reflection unless the Standard (Specular setup) shader is used.

**Comment by LocoCZ** [comment]
I might be wrong here about reflections, but isn't it possible to tick off the Reflections in the material setting for the tire? I have datamined core.masterbundle a few minutes back & toggling it off does make a change in Unity - not sure if it's going to make a change ingame though.
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/81763752/47414b8c-ca22-4acd-b1a5-4c0e18d540b5)

**Comment by SDGNelson** [comment]
The article is interesting - thanks! For the patch I've flipped all of the tread directions per your feedback. If I don't do it now it will never happen. :p

As for the reflections, I do wonder if we should get rid of the specular reflections entirely. In an update a year or two ago I removed them from all buildings. They're already disabled for foliage and the terrain as well IIRC. I haven't adjusted this in the patch, however.
